### ISSUE TITLE
Able Player button not centered in mobile WebKit browsers

### ISSUE THREAD

**Issue by conorom** [mlibrary/heliotrope#1439]
Although the tweak applied in #1401 got the play button to reappear in all WebKit browsers, it was no longer centered correctly. Or rather it was only centered in desktop Safari (larger iframes maybe?).

**Comment by conorom** [comment]
This issue caused me to revisit what I had tried initially which was to stop the big play button grabbing clicks by giving the now-superimposed controls a higher z-index. In a rush, I couldn't figure out why it wasn't working after multiple attempts, and ended up resizing and moving the play button out of the way instead. Much uglier.

After reading this article properly (alas, I did skim this page at some point before!) I see that giving the overlaid controls a z-index *without a non-default position* property can never work.  
https://philipwalton.com/articles/what-no-one-told-you-about-z-index/

At least reverting my former changes makes the Able-Player-related hacks simpler.
